### ISSUE TITLE
fs: re-evaluating native module sources is not supported

### ISSUE THREAD

**Issue by Odalrick** [commitizen/cz-cli#217]
I get the following error. Probably related to updating to node v6.0.0 .

$ git cz
(node:11355) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
/usr/lib/node_modules/commitizen/dist/commitizen/adapter.js:184
    throw error;
    ^

Error: Could not resolve /home/ulrik/projects/vertices-couchdb/src/db/fixtures/node_modules/cz-conventional-changelog
    at Function.Module._resolveFilename (module.js:438:15)
    at Function.resolve (internal/module.js:27:19)
    at resolveAdapterPath (/usr/lib/node_modules/commitizen/dist/commitizen/adapter.js:181:20)
    at gitCz (/usr/lib/node_modules/commitizen/dist/cli/strategies/git-cz.js:77:35)
    at Object.bootstrap (/usr/lib/node_modules/commitizen/dist/cli/git-cz.js:31:27)
    at Object.<anonymous> (/usr/lib/node_modules/commitizen/bin/git-cz.js:2:34)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)

**Comment by jimthedev** [comment]
We're unlikely to be able to do anything about this in the near term because it will take some time for our dependencies to upgrade to Node 6 compatibility.

We have several dependencies that use older version of graceful-fs. Take, for example gulp-git which relies on gulp which is not able to upgrade yet due to breaking changes. Until we can either move off of gulp-git for our commits OR we're able to move to a version of gulp-git that supports gulp 4.

You'll see quite a few similar threads if you look at other projects such as sequelize
https://github.com/sequelize/cli/issues/291

Or ionic:
https://forum.ionicframework.com/t/error-running-ionic-serve/50706

etc. etc.

Basically we're all keeping things steady until the changes can trickle down through the ecosystem. Everyone is doing their best to not break things for current users. This means patience while everyone works to update.

I would absolutely love if someone could put in a PR that allows us to stop using gulp-git and perhaps use a more streamlined wrapper. 

As soon as our dependencies are Node 6 compatible I will let you know. 

Thanks!

**Comment by Odalrick** [comment]
Good to know it isn't something _I_ broke.

**Comment by calvinmikael** [comment]
Is there a workaround for this besides downgrading to Node 5?

**Comment by jimthedev** [comment]
@calvinmikael Unfortunately not. We are still waiting on dependencies to upgrade to support node 6. I will try to put in some PRs with them to make sure it gets down to us but even then it might be a bit while things get merged. Thanks for your patience. Node 6 introduced a breaking change on a method used by a library used by most (if not all) of the filesystem watchers. We're in the same boat as them but I would hope that we can find a workaround soon.

**Comment by calvinmikael** [comment]
@jimthedev Thanks. I appreciate that you're trying to do the right thing here for everyone and it may take some time to get it right.

**Comment by jimthedev** [comment]
@calvinmikael @Odalrick Are you still seeing this? I get the warning but it commits just fine.

**Comment by jimthedev** [comment]
I've also blown away my node_modules directory and completely reinstalled commitizen and the adapter so perhaps an older version of graceful-fs is getting cached?

**Comment by calvinmikael** [comment]
@jimthedev Running git cz on Node 6.2.0 with a new node_modules directory I'm able to commit now.

**Comment by Odalrick** [comment]
@jimthedev Apart from the warnings it works on 6.2.0 now. I didn't even remove my node_modules.

**Comment by jimthedev** [comment]
@calvinmikael @Odalrick Great! Thanks for the update. The ecosystem comes through!

**Comment by Mbmaldon** [comment]
Hi i have problem with this error, (node:9112) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
